remindme-common/src/main/java/it/andreafailli/remindme/Profiles.java

```diff
@@ -0,0 +1,7 @@
+package it.andreafailli.remindme;
+
+public class Profiles {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)